pkg/agent/mcs.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package agent
+
+import (
+	"context"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	mcsclientset "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned"
+	mcsInformers "sigs.k8s.io/mcs-api/pkg/client/informers/externalversions"
+
+	"github.com/clusternet/clusternet/pkg/controllers/mcs"
+)
+
+type Mcs struct {
```

Comment:
It seems we could combine this file with `serviceexport_controller.go`.

Reply:
Later, we will include `endpointslice_controller`, so Mcs may be a wrapper to these two controllers, what do you think about that.